docs/guides-use-frame.md

```diff
@@ -4,10 +4,106 @@ title: How to use Frame
 sidebar_label: How to use Frame
 ---
 
-[//]: # "TODO: create guide"
-
-This guide help you get started with Frame to interact with your DAO. If you don't have a DAO yet create one following [your first aragon app tutorial](tutorial.md).
+This guide will help you to get started with Frame and interact with a DAO using a hardware wallet.
 
 ## Setup
 
-Start by downloading Frame from the official website: https://frame.sh.
+Start by downloading Frame from the official website: https://frame.sh. After installation Frame stays quietly in your menu bar until it's needed.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-intro.gif">
+</p>
+
+To take full advantage of Frame we recommend you have the `aragonCLI` installed as well. If you already have it, skip to the next section, otherwise ran:
```

Comment:
```suggestion
To take full advantage of Frame, we recommend you have `aragonCLI` installed as well. If you already have it, skip to the next section, otherwise run:
```